notebooks/official/automl/sdk_automl_tabular_regression_online_bq.ipynb

```diff
@@ -116,39 +116,6 @@
         "to generate a cost estimate based on your projected usage."
```

Comment:
<p>This should be using Vertex SDK for Python.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1246/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>

Reply:
<p>You could get model evaluations results using <code>model.list_model_evaluations()</code></p><p>As an example: </p><pre class="ql-syntax" spellcheck="false">if len(model.list_model_evaluations()) &gt; 0:
   # Parse evaluation data
   model_evaluations = 
        model.list_model_evaluations([0].to_dict()

   evaluation_metrics = model_evaluations["metrics"]


</pre>